dataprofiler/labelers/data_processing.py

```diff
@@ -1500,7 +1575,7 @@ def __eq__(self, other):
         :rtype: bool
         """
         if (
-            type(self) != type(other)
```

Comment:
why get rid of this check?

Reply:
I changed it to `not isinstance(other, StructCharPostprocessor)`, which allows `mypy` to infer the type of `other`. It should be equivalent.